setup.cfg

```diff
@@ -12,6 +12,10 @@ lint-reports = no
 lint-disable = locally-disabled,star-args
 lint-msg-template = {path}:{line}: [{msg_id}({symbol}), {obj}] {msg}
 
+[pylint.messages control]
```

Comment:
See #25 for a better fix

Reply:
Not sure, if I would call rewriting the whole `tox.ini` for a small lint-disable a "better fix"...

Reply:
Let's do the `tox` cleanup as a follow-up. Clean-start and all.

Reply:
#25 also contains the pylint checks (for a green CI)